proxies/live/apiproxy/policies/ServiceCallout.GetTargetsKvm.xml

```diff
@@ -4,7 +4,7 @@
   <Request>
     <Set>
       <Verb>GET</Verb>
-      <Path>NHSDigital/booking-and-referral-targets/master/targets/{{ APIGEE_ENVIRONMENT }}/targets.json</Path>
+      <Path>{bars_targets_path}</Path>
```

Comment:
We need the full URL here. This is the same for your infra PR as well. It should store the full url.